fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1656,21 +1656,17 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false,
     )
 
-    /**
-     * Indicates if juno onboarding feature is enabled.
-     */
-    val junoOnboardingEnabled: Boolean
```

Comment:
would this flag still be useful for dev purposes? E.g 
```
get() = !Config.channel.isDebug
```

Reply:
I don't believe so since the juno onboarding is already the default experience (eg, there's no alternative pathway).

Reply:
The behaviour I thought this would replicate (that is being removed) is how we currently disable it for dev with the manifest:
```
    defaults:
      - channel: developer
        value:
          enabled: false
```
Having to step through onboarding for each install may be a bit of a pain point?

Reply:
We spoke about this offline:

My take on this is that we should consider a secret setting pref that allows you to always see the juno onboarding on startup. My assumption is that it’s harder to get the juno onboarding showing consistently for debugging purposes compare to dismissing it entirely since you only ever have to run through the onboarding once. This variable where you have to manually modify, rebuild and launch doesn’t seem like the most developer/qa friendly approach. Having the secret settings would mean you would never have to rebuild and anybody including testers can always just toggle the juno onboarding without having to clear cache/storage/reinstall if they are testing the contents of the onboarding. You can still manually set the secret setting pref to false via as well.